auth/examples_test.go

```diff
@@ -49,38 +49,34 @@ func Example_serverConfig() {
 	)
 }
 
-type gRPCserverAuthenticated struct{}
-
-// SayHello only can be called by client when authenticated by exampleAuthFunc
-func (g gRPCserverAuthenticated) SayHello(ctx context.Context, request *pb.HelloRequest) (*pb.HelloReply, error) {
-	return &pb.HelloReply{Message: "pong authenticated"}, nil
+type server struct {
+	pb.UnimplementedTestServiceServer
+	message string
 }
 
-type gRPCserverUnauthenticated struct{}
-
-// SayHello can be called by client without being authenticated by exampleAuthFunc as AuthFuncOverride is called instead
-func (g *gRPCserverUnauthenticated) SayHello(ctx context.Context, request *pb.HelloRequest) (*pb.HelloReply, error) {
-	return &pb.HelloReply{Message: "pong unauthenticated"}, nil
+// SayHello only can be called by client when authenticated by exampleAuthFunc
```

Comment:
Let's stick to https://thanos.io/tip/contributing/coding-style-guide.md/#comments-should-be-full-sentences rule 🤗

Reply:
Not a blocker, so let's merge.